sdk/py/apepay/manager.py

```diff
@@ -89,21 +91,32 @@ def _parse_validator(self) -> Callable[[_ValidatorItem], Validator]:
     def set_validators(self) -> ContractTransactionHandler:
 
         @wraps(self.contract.set_validators)
-        def order_validators(*validators: _ValidatorItem, **txn_kwargs) -> ReceiptAPI:
+        def set_validators(*validators: _ValidatorItem, **txn_kwargs) -> ReceiptAPI:
             # NOTE: Always keep sets sorted, ensure no duplicates
-            return self.contract.set_validators(
-                sorted(v.address for v in set(map(self._parse_validator, validators))),
-                **txn_kwargs,
-            )
+            new_validators = sorted(v.address for v in set(map(self._parse_validator, validators)))
+            # NOTE: Okay to use Click here, intended primarily to support interactive features
```

Comment:
Does this comment mean you shouldn't use this method in a regular library sort of way?

Reply:
You can, it would just probably not be common and you would get some extra printout happening

Really, I just couldn't figure out the best way to use Ape's logger via `ManagerAccessMixin`

Reply:
typically you import it `from ape.logging import logger`

Reply:
Actually another under-rated approach is subclass `ApeCliContextObject` especially if this class is designed for CLI usage only
And you can use it in your CLIs like:

```python
@ape_cli_context(obj_type=Runner)
```